app/src/main/res/values/strings.xml

```diff
@@ -1093,5 +1093,6 @@
     <string name="no_data_fetched">No Data Fetched</string>
     <string name="recorder">Recorder</string>
     <string name="save_graph">Save Graph</string>
-
+    <string name="delete_all">Delete All Data</string>
+    <string name="delete_all_message">Are Sure You want Delete All Logged Data?</string>
```

Comment:
@neel1998 Follow standard casing of letters. Else the PR looks good.

Reply:
you mean make the 'w' of 'want' upper case, right?

Reply:
Change it too - "Are you sure wanna delete all logged data?" Also change the line above it.

Reply:
Aren't we keeping first character of each word in upper case through out the project?

Reply:
@harsh-2711  ?

Reply:
Oops. We need to fix that issue. Sentences need to be grammatically correct. Could you work on thay @neel1998?

Reply:
@CloudyPadmal  so the string should be "Are You Sure You Want To Delete All Logged Data?"

Reply:
@harsh-2711  can you please explain what exactly needs to be changed so that this PR can be closed

Reply:
"Are you sure?"

Is enough.